src/app/reporting/Engine.h

```diff
@@ -77,6 +79,11 @@ class Engine
      */
     CHIP_ERROR ScheduleRun();
 
+    /**
+     * Application marks mutated change path and would be sent out in later report.
+     */
+    CHIP_ERROR SetDirty(ClusterInfo & aClusterInfo);
```

Comment:
Inconsistent naming with `mpChangePathList`.

Reply:
Yep, update it with mpGlobalDirtyList